source/styles.json

```diff
@@ -0,0 +1,66 @@
+{
+  "rows": {
+
+            "numRows":"3",
+            "translation": "[50, 150]",
+            "itemSize": "[ 2000, 350 ]",
+            "showRowLabel": "[ true ]",
+            "rowLabelOffset": "[ [0, 15], [0, 15], [0, 15] ]",
+            "drawFocusFeedback":true,
+            "vertFocusAnimationStyle":"fixedFocusWrap",
+            "rowFocusAnimationStyle": "fixedFocusWrap",
+            "visible": true,
+            "rowItemSize":"[[500, 285]]",
+            "itemSpacing":"[0, 80]",
+            "rowItemSpacing": " [[30, 0]]"
+          },
+
+  "video":{
+
+              "width":"1920",
+              "height":"1080",
+              "visible":false
+          },
+
+  "detail":{
```

Comment:
same here.. ._"detail"_ is too abstract, "detail" of what?

Reply:
It's the detail of the video. Changed.